### ISSUE TITLE
Extraction failing on all dinos.

### ISSUE THREAD

**Issue by Dean050** [cadon/ARKStatsExtractor#289]
Simply put, all the multipliers and stat settings are defaulted as I'm on official, everything is set correct. Yet everytime I try to read values it throws out the wrong statistics for my wild dinos im attempting to tame, it throws out bad values for my tamed dinos, and it throws out errors for any breeds. It's worked in the past when I played a couple months back, but after recently coming back to ARK and trying the tool again it never seems to work. 

Here is a [screenshot](https://gyazo.com/6d19e9acb2d5895cbb2fd9fea285780b) as an example, I can get some more tomorrow of various dinos if needed.

**Comment by cadon** [comment]
Hi, there are some typos in the values in the extractor. The new UI makes it harder for the OCR to read all values correctly, so always double-check and correct them manually if an extraction fails.

It seems the stamina is very high, probably this value is from before the recent (~ a month ago) flyer-nerf, that decreased that for most birds. Probably leveling up stamina will recalculate with the new values.

**Comment by Dean050** [comment]
Yes, the values its taken by reading from the ark window are totally wrong, putting them manually doesn't help.  I cannot read values at all from any dinos, be it tamed or bred.
  I use this tool a lot before to figure out the post tame base stats on bred dinos that have been leveled, to keep a library of my tames with their base stats and added levels, and bred ones. 
  Now I can't do any of that do to random stuff being inputted by reading from the ark window.   Leveling the stamina on that example didn't help either. It's not just flyers, it's all dinos. 

In these screenshots you, I select the dino type and read, this is pretty much what happens. All of the values are wrong and random. It makes no sense. Like I said before, I've used the program extensively in the past, I stopped playing in december, come back and these new versions dont work for me at all
https://gyazo.com/6ac4064b3e935fdd0c1573ea94456f27
https://gyazo.com/2b4e01c9ce4c4bf46c0ef1e239f1c2ee
